docs/cookbook/customization/form.rst

```diff
@@ -0,0 +1,115 @@
+Customizing Forms
+=================
+
+The forms in Sylius are placed in the ``Sylius\Bundle\*BundleName*\Form\Type`` namespaces.
+
+.. warning::
+    Many forms in Sylius are **extended in the CoreBundle**.
+    If the form you are willing to override exists in the ``CoreBundle`` your should be extending the one from Core, not the base form from the bundle.
+
+Why would you customize a Form?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are plenty of reasons to modify forms that have already been defined in Sylius.
+Your business needs may sometimes slightly differ from our internal assumptions.
+
+You can:
+
+* add completely **new fields**, if you need another phone number for your customers,
+* **modify** existing fields, change labels, make required, change choice types etc.,
```

Comment:
`(...) existing fields, make them required, change their HTML class, change labels and so on/etc/spaghetti` wdyt?